src/components/CurrentPhoto/CurrentPhoto.jsx

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './CurrentPhoto.css';
 
 const CurrentPhoto = (props) => {
   return (
-    <div> CurrentPhoto Component </div>
+    <div className={styles['main-image-box']}>
+      <img style={{ maxWidth: '100%' }} src={`${props.currentPhoto}`} alt="room" />
```

Comment:
For the `src` value, probably don't need the template literal?

Reply:
Absolutely.  I just removed it.